packages/primitives-core/README.md

```diff
@@ -0,0 +1,39 @@
+# @emotion/primitives-core
+
+> Shared utilities for emotion primitives and native
+
+## Introduction
+
+This package contains some shared utilities which are consumed by `@emotion/primitives` and `@emotion/native` for styling and rendering components.
```

Comment:
Could you add a note saying that most people won't need this package and should use `@emotion/primitives` and `@emotion/native`?